integreat_cms/cms/views/dashboard/dashboard_view.py

```diff
@@ -86,7 +85,7 @@ def get_hix_context(self):
         ready_for_mt = math.trunc(100 * ready_for_mt_count / len(hix_translations))
 
         return {
-            "page_translations": worst_his_translations,
+            "page_translations": worst_hix_translations,
```

Comment:
```suggestion
            "worst_hix_translations": worst_hix_translations,
```
I think it makes sense to keep variable names consistent in the Python views and the template code, so it's a bit easier to track down where a variable is coming from.